### DISCUSSION TITLE
【求助】绳网是不是需要再优化一下，还是说是我的问题

### DISCUSSION THREAD

**Discussion by AKT505** [share121/inter-knot#460] [其他]
大家逛绳网的时候终端什么状况，我的终端刷个几分钟就热过头开始穿黑丝女仆装了。
![553f77b131ef5b9af9b513487642f346](https://github.com/user-attachments/assets/e2d7ce3e-256c-4462-a431-f51243dcce1d)

**Comment by huashuideryu** [comment]
建议发先我看看[doge]

**Comment by SaKura-071721** [comment]
我看你是缺猫娘了
